Documentation/configuration/metrics.rst

```diff
@@ -125,7 +125,10 @@ SubProcess
 Kubernetes
 -----------
 
-* ``kubernetes_events_total``: Number of Kubernetes events received labeled by
+* ``kubernetes_events_received_total``: Number of Kubernetes events received labeled by
+  scope, action, validity and equality
+
+* ``kubernetes_events_total``: Number of Kubernetes events processed labeled by
```

Comment:
Should this be renamed to ``kubernetes_processed_events``? Not sure how renaming an already existing metric would work, though :/

Reply:
We would need to add a new metric and obsolete the old one. That's why I did not rename it.